spec/schemas/MerkleInfo.yml

```diff
@@ -0,0 +1,6 @@
+type: object
```

Comment:
No `description` here? What is this string?

Reply:
This is the missing part. I still don't know how to parse the server API merkle path. Once I have it this object will be modified and documented.

Reply:
👍

Reply:
Hi @segfaultxavi . I've updated the information with the best information I have right now. In the future, we will upgrade this object adding a better structure for state proof.

Reply:
Hi @segfaultxavi  the raw string is the best I can return right now. In the future, the object will also have a list describing the nodes in the tree. Something like: 

```
[
  { type: branch, path: ..., linkMask: xxx, links: [ ... , ... ] },
  { type: leaf, path: ..., hash }
]
```